tests/FactoryTest.php

```diff
@@ -198,6 +199,20 @@ public function testParseEmailAddresses()
         $this->assertEquals($expected, $factory->parseEmailAddresses($addresses));
     }
 
+    public function testNineChildParts()
```

Comment:
Would be nice to add a comment to explain this is testing the check for *1.10* doesn't fail.